filter_lists/regional.json

```diff
@@ -362,7 +362,7 @@
     {
         "uuid": "A0E9F361-A01F-4C0E-A52D-2977A1AD4BFB",
         "url": "https://raw.githubusercontent.com/gioxx/xfiles/master/filtri.txt",
-        "title": "ABP X Files",
+        "title": "ABP X",
```

Comment:
- I think `Files` is worth including here
- Also, the "official" title of this one is `NoAds X Files`; not sure if that's a recent change but we should probably follow
- It'd be good to distinguish this as a primarily Italian list too... maybe `NoAds X Files Italian`?